handlers/whatsapp/whatsapp.go

```diff
@@ -436,7 +436,7 @@ type Param struct {
 
 type Component struct {
 	Type       string  `json:"type"`
-	Parameters []Param `json:"parameters"`
```

Comment:
or we can make sure this is initialized to an empty array so sends `[]`

Reply:
Yeah, I had thought of that but went with interpreting optional as something that is present or not